addons/sale/views/sale_views.xml

```diff
@@ -1228,7 +1228,13 @@
             <field name="arch" type="xml">
                 <data>
                     <xpath expr="//field[@name='partner_id']" position="after">
-                       <field name="partner_shipping_id"
+                        <div class="o_td_label" groups="sale.group_delivery_invoice_address">
+                            <label for="partner_shipping_id" string="Delivery Address" style="font-weight:bold;"
+                                   attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund'))]}"/>
+                            <label for="partner_shipping_id" string="Dispatch Address" style="font-weight:bold;"
```

Comment:
Yes, but I can't find any xpaths that using partner_shipping_id for account.move

Reply:
I'm sorry but the `partner_shipping_id` should be invisible on vendor bill because:
- Added by the `sale` module
- The fact it's currently visible is an unexpected behavior
- There is no management anywhere for this field on vendor bill
- It's introducing confusion for customers, see https://www.odoo.com/web#id=2668902&cids=1&model=project.task&view_type=form

Reply:
@smetl right we not have any management for vendor bill but can we just simple fix this string and remove confusion in V15 and add management for this in master.

Reply:
@smetl I don't agree.  It is not because the field is in sale that it does not make any sense in a vendor bill.  There is a reason for moving it to account as well.  

The goal of Jigar is to be clear with dispatch address, because that is a really useful thing.

Reply:
Yes but you will not move this field in a stable version. So instead of introducing unexpected things in stable, hide it for vendor bill and handle the field properly in master

Reply:
Why would we hide it if it serves a purpose?  Maybe we can just improve the help= then  (this thing has been like this for a while anyways) @smetl